### PR TITLE
Minor README fix

### PR DESCRIPTION
**Pull request by Jonathan-Rosenberg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ file system](#default-file-system) for the URI scheme.
 
 The final configuration step is to tell Spark what file system to use for each URI scheme. You should make sure that you 
 add this configuration for any URI scheme you defined a mapping configuration for.
-For example, to tell Spark to use `S3AFileSystem` for any URI with `scheme=s3a`
+For example, to tell Spark to use `S3AFileSystem` for any URI with `scheme=lakefs`
```
Comment (lynnro314) [inline README.md:74]: Maybe:
```suggestion
For example, tell Spark to use `S3AFileSystem` for any URI with: `scheme=lakefs`
```

**Review by lynnro314 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # hadoop-router-fs
 
-[RouterFileSystem](src/main/java/io/lakefs/RouterFileSystem.java) is a Hadoop [FileSystem](https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/fs/FileSystem.html) 
+[RouterFileSystem](src/main/java/io/lakefs/RouterFileSystem.java) is a Hadoop [FileSystem](https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/fs/FileSystem.html) class
```
Comment (arielshaqed) [inline README.md]: I think I prefer the old one.  Spark users see it as a "FileSystem" they can use.  Hadoop has mixed usages, but usually just calls it "S3AFileSystem" (without "class"). The main place I found that does say "S3AFileSystem class" was in its [configuration guide](https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/index.html#general_configuration):

> ```xml
> <property>
>   <name>fs.s3a.impl</name>
>   <value>org.apache.hadoop.fs.s3a.S3AFileSystem</value>
>   <description>The implementation class of the S3A Filesystem</description>
> </property>
> ```

But here it actually _does mean_ "class".

Think of a (very...) naive PySpark or Spark-SQL user.  To them, "class" is an implementation detail _on a distant part of the system_.

File: README.md (hunk 1 above)
Comment (arielshaqed) [inline README.md:74]: I would rather avoid "lakefs" here entirely.  Perhaps "minio"?

**Review by arielshaqed — COMMENTED** [review]
Not sure about both of these changes; made a few suggestions...

**Pull request merged**